form/index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/css/global.css" />
+    <script src="/js/index.js" defer></script>
+    <title>Quiz App - Form</title>
+  </head>
+  <body>
+    <header>
+      <h1>Quiz App</h1>
+    </header>
+    <main class="main__form">
+      <h2 class="form-title" id="form-title">Create a new card</h2>
+      <p class="form-decription" id="form-description">
+        Please fill out this form to create a new card.
+      </p>
+      <section class="form-container">
+        <form
+          class="form"
+          data-js="form"
+          aria-labelledby="form-title"
+          aria-describedby="form-description"
+        >
+          <label class="label" for="form-cardQuestion">Your question:</label>
+          <textarea name="form-cardQuestion" rows="3"></textarea>
+          <label class="label" for="form-cardAnswer">Your answer:</label>
+          <textarea name="form-cardAnswer" rows="3"></textarea>
+          <label for="form-cardTag">Tag:</label>
+          <input type="text" name="form-cardTag" />
+
+          <button
+            type="submit"
+            class="card__button"
+            name="submit-new-card"
```

Comment:
I think the `name` attribute is not necessary  here (does a button has a name attribute?)